frontend/packages/console-app/src/components/volume-snapshot/create-volume-snapshot/create-volume-snapshot.tsx

```diff
@@ -170,11 +160,38 @@ const CreateSnapshotForm = withHandlePromise<SnapshotResourceProps>((props) => {
   }, [namespace, pvcName]);
 
   const [data, loaded, loadError] = useK8sWatchResource<PersistentVolumeClaimKind[]>(resourceWatch);
+  const scList = scObjListLoaded ? (scObjList.items as StorageClassResourceKind[]) : [];
```

Comment:
you dont have to type cast. `scObjList.items` should be properly typed already

Reply:
done